_posts/2024-05-03-gsoc-outreachy-students.md

```diff
@@ -25,17 +25,15 @@ Add a link to your blog here!
 ## Valerij Talagayev: [2D Visualization for Small Molecules](https://summerofcode.withgoogle.com/programs/2024/projects/sfy3kuqc)
 
 <img
-src="Picture Here"
+src="https://avatars.githubusercontent.com/talagayev"
 title="Valerij Talagayev" alt="Valerij Talagayev"
 style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
 
-Write a short description of your project here!
-
-Write something about yourself here! (affiliation, background, interests - work and/or personal, etc.)
+2D Visualization of small molecules is quite a helpful way to recognize the different parts of the small molecule. The main objective of Valerij's GSOC project will revolve around the 2D visualization of small molecules through the application of the MDAnalysis RDKit converter, which would allow to convert ligands into RDKit objects and thus allow the user to look at the ligand in his MDAnalysis Universe in an easy to use interactive display in jupyter notebook.
```

Comment:
```suggestion
2D Visualization of small molecules is quite a helpful way to recognize the different parts of the small molecule. The main objective of Valerij's GSoC project will revolve around the 2D visualization of small molecules through the application of the MDAnalysis RDKit converter, which would allow to convert ligands into RDKit objects and thus allow the user to look at the ligand in his MDAnalysis Universe in an easy to use interactive display in [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/).
```